torch/distributed/rpc.py

```diff
@@ -79,6 +79,7 @@ def _init_rpc(backend=RpcBackend.PROCESS_GROUP,
     elif is_rpc_backend_registered(backend):
         _agent = init_rpc_backend(
             backend,
+            self_rank=self_rank,
```

Comment:
Can you add a test case where you register a backend and call init ?

Reply:
@satgera working on it. It requires me to patch init_rref_context